src/main/java/org/folio/rest/impl/ServicePointAPI.java

```diff
@@ -326,6 +349,23 @@ public void putServicePointsByServicepointId(String servicepointId,
     });
   }
 
+  public void putServicePoints(Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext){
```

Comment:
Uh... What's this all about? :) Can we remove this? I don't see how the RAML would generate a PUT for this resource. If it does, we should probably fix it so it doesn't. I'm not sure what PUT without an ID means in REST.

Reply:
Removed